web/pdf_page_view.js

```diff
@@ -1090,9 +1097,13 @@ class PDFPageView {
           viewport.rawDims
         );
 
-        this.#renderTextLayer();
+        const textLayerP = this.#renderTextLayer();
```

Comment:
No single letter abbreviations please.
```suggestion
        const textLayerPromise = this.#renderTextLayer();
```